Riot/AppDelegate.m

```diff
@@ -2261,6 +2261,7 @@ - (void)dismissCallViewController:(MXKCallViewController *)callViewController co
             _incomingCallNotification = nil;
             
             // Release properly
+            [currentCallViewController destroy];
             currentCallViewController.mxCall.delegate = nil;
```

Comment:
Thanks to the destroy call, the 2 following lines are not relevant anymore
 ``` currentCallViewController.mxCall.delegate = nil; currentCallViewController.delegate = nil; ```
Could you please remove them?